### DISCUSSION TITLE
Nuxt tag in feed

### DISCUSSION THREAD

**Discussion by dbaranec** [dailydotdev/daily#1233] [Feature Requests]
Please add a Nuxt tag so that it can be selected when someone is creating a feed, and they would receive articles about Nuxt.
Currently, it is possible to choose React -> Next,  Vue –> but Nuxt is missing.

**Comment by idoshamun** [comment]
Hi @dbaranec,

Just did. Here you go: https://app.daily.dev/tags/nuxt
Enjoy!
